README.md

```diff
@@ -71,6 +71,10 @@ Apache Polaris is built using Gradle with Java 21+ and Docker 27+.
 - `./gradlew run` - To run the Polaris server locally; the server is reachable at 
   localhost:8181. This is also suitable for running regression tests, or for connecting with Spark. 
 - `./regtests/run_spark_sql.sh` - To connect from Spark SQL. Here are some example commands to run in the Spark SQL shell:
+- Set your own credentials using the environment variable `export POLARIS_BOOTSTRAP_CREDENTIALS=POLARIS,root,s3cr3t` where:
```

Comment:
I think it might be best to disambiguate by saying `If you start Polaris as a direct `java` process with in-memory storage, set you own credentials ...`?

Reply:
I assume this is only for `./gradlew run`. Can we move it under that bullet point so it's clear this environment variable (`POLARIS_BOOTSTRAP_CREDENTIALS`) is specifically for running Polaris locally via the Gradle task?